### ISSUE TITLE
Linear SVM with SGD Training Support

### ISSUE THREAD

**Issue by crjackso** [accord-net/framework#213]
Hello,

 Is it possible with Accord to create a linear SVM classifier with Stochastic Gradient Descent (SGD) training?  I see there is a LogisticGradientDescent class, but it looks like this is for logistic regression, not SVM.

Thanks!

```
var logisticGradientDescent = new LogisticGradientDescent(new LogisticRegression(featureCount))
                {
                    Stochastic = true,
                    LearningRate = 0.01
                };
```

**Comment by cesarsouza** [comment]
While I had initially scheduled this feature to an earlier release, I couldn't manage to work on it yet. Sorry about that. 

If anyone would like to adventure in implementing one of those algorithms, please feel free to proceed. As I said before, they are really simple to implement (SGD updates can be implemented in one single for loop, and the gradient updates for SVMs are readily available online). It should also be possible to directly port it from Python (e.g. from [sklearn](https://github.com/scikit-learn/scikit-learn/blob/14031f6/sklearn/linear_model/stochastic_gradient.py#L548) and other libraries that have permissive compatible licenses).

It seems a good source for inspiration (if its license allow it), is http://leon.bottou.org/projects/sgd

**Comment by cesarsouza** [comment]
Fixed in 3.5.0.
